index.html

```diff
@@ -41,18 +41,33 @@
             <div class="eight columns">
                 <h1 id="about">About</h1>
                 <p>
-                    TotalityHacks was born by asking what would happen if we could design a hackathon from scratch. This re-design started with the organizing team: rather than being organized by students of a singular university and hosted at that university, TotalityHack's team comes from a wide array of schools across the country. The advantages of this are countless, but as an attendee the biggest difference is focus. Instead of needing to try and cater to beginners from the host school and experienced hackers who travel in, we're choosing to just cater TotalityHacks to the most committed and experienced hackers. 
+                    TotalityHacks is a new kind of hackathon for students–rather than being organized by students of a singular 
+                    university and hosted at that university, TotalityHack's team comes from a wide array of schools across 
+                    the country. The advantages of this are countless, but as an attendee the biggest difference is focus. 
+                    Instead of needing to try and cater to beginners from the host school and experienced hackers who travel 
+                    in, we're choosing to just cater TotalityHacks to the most committed and experienced hackers. Our team is 
+                    seasoned in organizing and participating hackathons, so we will make sure your experience at 
+                    TotalityHacks will be the best one to date.
```

Comment:
The "TotalityHacks will the best one to date" bothers me just a little. Not sure why, but it doesn't sound compelling at all.

Reply:
Agree with @gmittal. I also think the "our team is seasoned" seems like a bit too hard of a sell so I'd probably remove that sentence all together.